superset/assets/src/components/FilterableTable/FilterableTableStyles.css

```diff
@@ -72,3 +72,10 @@
 }
 .even-row { background: #f2f2f2; }
 .odd-row { background: #ffffff; }
+.header-style {
+  direction: rtl;
```

Comment:
Do we need this? Superset supports different translations, and potentially some of them could be `rtl`.

Reply:
Good point. Maybe I should just make it the standard ellipsis for now to be consistent with the rest of the app because the answer to your question should be implemented throughout the product.